cpp/ycm/ClangCompleter/Diagnostic.h

```diff
@@ -33,6 +33,34 @@ enum DiagnosticKind {
   WARNING
 };
 
+/// Information about a replacement that can be made to the source to "fix" a
+/// diagnostic
+struct FixItChunk {
+  /// the replacement string. this string should replace the source range
```

Comment:
Please use capital letters for sentences in comments; it's extra jarring when you have periods etc. :)